src/edfi-performance-test/edfi_performance_test/factories/resources/student.py

```diff
@@ -28,73 +30,9 @@ class StudentFactory(APIFactory):
     lastSurname = "Jones"
     generationCodeSuffix = "JR"
     personalTitlePrefix = "Mr"
-    hispanicLatinoEthnicity = False
```

Comment:
Someone must have been looking at the Data Standard 2.x definition instead of 3.x when writing this!